modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -86,7 +88,7 @@ spec:
      - secureforward-offcluster
 ----
 <1> The name of the log forwarding CR must be `instance`.
-<2> Parameter to enable log forwarding. Set to `true` to enable log forwarding.
+<2> Parameter to disable the default log forwarding behavior. Set to `true` to enable log forwarding to the specified outputs.
```

Comment:
This switch does not explicitly enable `enable log forwarding to the specified outputs`; that is what the annotation essentially does.  This switch only *disables* the operator from automatically forwarding logs to its own managed logstore